datadog_checks_base/datadog_checks/base/log.py

```diff
@@ -74,15 +73,30 @@ def getEffectiveLevel(self):
 
 
 class CheckLogFormatter(logging.Formatter):
+    def __init__(self):
+        super(CheckLogFormatter, self).__init__()
+        integration_tracing, _ = tracing_enabled()
+        self.integration_tracing_enabled = integration_tracing
+
     def format(self, record):
         # type: (logging.LogRecord) -> str
         message = to_native_string(super(CheckLogFormatter, self).format(record))
-        return "{} | ({}:{}) | {}".format(
+
+        attributes = {
             # Default to `-` for non-check logs
-            getattr(record, '_check_id', '-'),
-            getattr(record, '_filename', record.filename),
-            getattr(record, '_lineno', record.lineno),
-            message,
+            'check_id': getattr(record, '_check_id', '-'),
+            'filename': getattr(record, '_filename', record.filename),
+            'lineno': getattr(record, '_lineno', record.lineno),
+            'message': message,
+        }
+
+        if not self.integration_tracing_enabled:
```

Comment:
Can you please keep the existing logic by putting this check first and then adding the new logic below that?

Reply:
Sure, updated